docs/content/docs/development/streaming-query.md

```diff
@@ -75,14 +75,14 @@ streaming mode. This mode has a lower cost compared to Kafka but has a higher la
 depending on the checkpoint interval of the writing job.
 
 By default, the downstream streaming consumption is disordered (ordered within the key)
-stream of upsert data. If you expect an ordered CDC data stream, you can configure it
-as follows (recommended):
+stream of upsert data. If you expect an ordered CDC data stream, and remove downstream
```

Comment:
I feel it is a little hard to understand from a user's perspective. A more intuitive way is to add an example and some pic(s) to illustrate.

For example, we can take `ChangelogWithKeyFileStoreTableTest#testStreamingChangelog` to explain the final result difference between enable/disable `ChangelogProducer.INPUT`

And meanwhile, add some pic to illustrate the processing difference

Reply:
How about
![image](https://user-images.githubusercontent.com/55568005/181272707-e1b06183-2c6b-454a-bedf-aee9e1504de9.png)

![image](https://user-images.githubusercontent.com/55568005/181272515-c579095d-6e8e-42fb-b4f5-6d13aaa10cd7.png)